client/app/actions/RoomActions.js

```diff
@@ -1,33 +1,11 @@
 'use strict';
 
-var RoomConstants = require("../constants/RoomConstants.js");
-var AppDispatcher = require("../dispatcher/AppDispatcher.js");
+var TuxActions = require("../../../node_modules/tux/Actions");
```

Comment:
Can just `require('tux/Actions') even though it hasnt been npm installed.  `Browserify` defaults to searching through `node_modules` so as long as `tux` is located there it'll find it.  Apply to all other `require` statements.